src/reducers/index.js

```diff
@@ -1,4 +1,7 @@
-import { combineReducers } from 'redux';
+import { combineReducers, createStore } from 'redux';
 import booksReducer from './books';
 
-export default combineReducers({ books: booksReducer });
+const store = createStore(
+  combineReducers({ books: booksReducer }),
+);
```

Comment:
I think your store should be created in `src/index.js` from milestone 2 requirement.

Reply:
Fixed this issue.